src/app.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Form from './form'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
```

Comment:
This `state` (question and answer) should be located inside your form component.